http/http-api/src/main/java/com/okta/commons/http/MimeType.java

```diff
@@ -422,7 +422,9 @@ private boolean parametersAreEqual(MimeType other) {
             return false;
         }
 
-        for (String key : this.parameters.keySet()) {
+        for (Map.Entry<String, String> entry : this.parameters.entrySet()) {
```

Comment:
Fixes below findbugs error:

` Redundant nullcheck of str, which is known to be non-null in com.okta.commons.lang.Objects.nullSafeToString(Object) [com.okta.commons.lang.Objects] Redundant null check at Objects.java:[line 664] RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE`